cli/deploy-profiling-job.sh

```diff
@@ -0,0 +1,123 @@
+## IMPORTANT: this file and accompanying assets are the source for snippets in https://docs.microsoft.com/azure/machine-learning! 
+## Please reach out to the Azure ML docs & samples team before before editing for the first time.
+
+## Preparation Steps:
+## 1. az upgrade -y
+## 2. az extension remove -n ml
+## 3. az extension remove -n azure-cli-ml
+## 4. az extension add -n ml
+## 5. az login
+## 6. az account set --subscription "<YOUR_SUBSCRIPTION>"
+## 7. az configure --defaults group=<RESOURCE_GROUP> workspace=<WORKSPACE_NAME>
+
```

Comment:
name of this .sh file is mis-leading: no deploy concept for job, and there's no `profiling job`. considering rename to `profile online endpoint` or `how to profile online endpoint`. 

such concept alignment within v2 is critical to reduce onboarding cost and make the sell easier.

Reply:
file name changed to how-to-profiling-online-endpoint.sh